azure-kusto-data/azure/kusto/data/asyncio/utils.py

```diff
@@ -0,0 +1,9 @@
+import aiohttp
+
+
+async def run_request(endpoint: str, **kwargs) -> dict:
```

Comment:
I dislike utils classes. I prefer you put this function close to it's usage (kusto doesn't provide utils)

Reply:
👍